README.md

```diff
@@ -46,6 +46,28 @@ The last thing is to change the training and model code slightly to process your
 
 Then you should be able to run your training code.
 
+Trained Model
+-------------
+We also porvide a model which is already trained on STS dataset. So it is easier if you just want to use the model and do not want to re-train it. 
+
+The tarined model download link is [HERE](https://drive.google.com/file/d/0B-lu_eEMkpVxYVdPMldJX3JDVjg/view?usp=sharing). Model file size is 500MB. To use the trained model, then simply use codes below:
+```
+include('Conv.lua')
```

Comment:
/home/tilakputta/torch/install/bin/luajit: /home/tilakputta/torch/install/share/lua/5.1/torch/init.lua:102: while creating metatable similarityMeasure.Conv: bad argument #1 (similarityMeasure is an invalid module name)
stack traceback:
	[C]: in function 'newmetatable'
	/home/tilakputta/torch/install/share/lua/5.1/torch/init.lua:102: in function 'class'
	...kputta/Desktop/sts/textSimilarityConvNet-master/Conv.lua:1: in main chunk
	[C]: in function 'dofile'
	/home/tilakputta/torch/install/share/lua/5.1/torch/init.lua:49: in function 'include'
	test.lua:1: in main chunk
	[C]: in function 'dofile'
	...utta/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x00405d50